engine/runtime/src/main/java/org/enso/interpreter/runtime/data/atom/SuspendedFieldGetterNode.java

```diff
@@ -33,7 +36,51 @@ private SuspendedFieldGetterNode(
     this.set = set;
   }
 
-  static UnboxingAtom.FieldGetterNode build(
+  static NodeFactory<? extends UnboxingAtom.FieldGetterNode> factory(
```

Comment:
Wrapping of `FieldGetterNode` now happens on the level of _factories_. As such it should be more consistent and the _suspended wrapper_ shall be used everywhere.